src/renderer/components/+catalog/catalog.tsx

```diff
@@ -44,7 +44,8 @@ import { CatalogMenu } from "./catalog-menu";
 import { HotbarIcon } from "../hotbar/hotbar-icon";
 import { RenderDelay } from "../render-delay/render-delay";
 
-export const previousActiveTab = createAppStorage("catalog-previous-active-tab", "");
+export const browseCatalogPage = "browse";
```

Comment:
Why this constant is here? Should we keep all known groups for catalog in `routes/catalog.ts`?

Reply:
Moving this to `/common/routes/catalog.ts`. All of the other groups are dynamic and generated on demand from `catalogCategoryRegistry`.